app/models/concerns/fitness.rb

```diff
@@ -36,6 +36,14 @@ def self.developers_fitness(contractor_opts)
 
 private
 
+  def self.lin_to_root(x)
```

Comment:
`private` (on line 37) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.